components/PlanetIllustrations.js

```diff
@@ -0,0 +1,18 @@
+export function PlanetIllustrations({ hiddenOnMobile }) {
+    return (
+        <div className="absolute w-full h-screen">
+            <div className="absolute hidden w-[243px] h-[243px] top-16 left-48 sm:block">
+                <img src="/planet-top-left.svg" alt="planet1" />
+            </div>
+            <div className={`absolute right-0 sm:w-[297px] sm:h-[347px] sm:-top-4 sm:right-0 w-[227px] h-[257px] ${hiddenOnMobile && "hidden sm:block"}`}>
```

Comment:
Is there any need for mobile check with 'hiddenOnMobile' when you set 'hidden sm:block' already @egebircan

Reply:
We talked it with @egebircan